src/Skia/Perspex.Skia.Android/AndroidRenderTarget.cs

```diff
@@ -14,6 +14,8 @@
 
 namespace Perspex.Skia
 {
+/* This lives in shared project now, but we may need an android specific implementation anyways
```

Comment:
I'd prefer to remove commented-out code like this before merging to master.

In my current job I inherited a project that is about 15 years old that was *full* of commented-out code, and it made the codebase incredibly difficult to understand because the comments were at times 10 years out of date, and it wasn't clear *why* the code was commented out. I'd rather that not happen to Perspex ;)